extension.ts

```diff
@@ -19,15 +19,51 @@ export function activate(context: vscode.ExtensionContext) {
     let projectsStored: string = context.globalState.get<string>('recent', '');
     let aStack: stack.StringStack = new stack.StringStack();
     aStack.fromString(projectsStored);
+    showStatusBar();
     
     // register commands
     vscode.commands.registerCommand('projectManager.saveProject', () => saveProject());
     vscode.commands.registerCommand('projectManager.listProjects', () => listProjects(false));
     vscode.commands.registerCommand('projectManager.listProjectsNewWindow', () => listProjects(true));
     vscode.commands.registerCommand('projectManager.editProjects', () => editProjects());
-    
+
+    function showStatusBar(projectName?: string) {
+
+        let statusItem = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Left);
+        // if we have a projectName, we don't need to search. 
+        if (projectName) {
+            statusItem.text = projectName;
+            statusItem.show();
+            return;
+        }
+
+        let currentProjectPath = compactHomePath(vscode.workspace.rootPath);
+
+        let items = []
+        if (fs.existsSync(getProjectFilePath())) {
+            items = loadProjects(getProjectFilePath());
+            if (items == null) {
+                return;
+            }
+        }
+
+        let foundProjectLabel: string = null;
+        for (let i = 0; i < items.length; i++) {
+            let element = items[i];
+            if (element.description == currentProjectPath) {
```

Comment:
Beware with _case sensitive_. It's better to use `element.description.toString().toLowerCase() == currentProjectPath.toLowerCase()`